dubbo-samples-protosubff/dubbo-samples-protosubff-api/src/main/java/org/dubbo/samples/protosubff/patch/ProtosubffPatch.java

```diff
@@ -0,0 +1,103 @@
+package org.dubbo.samples.protosubff.patch;
+
+import java.io.IOException;
+
+import org.apache.dubbo.common.Version;
+import org.apache.dubbo.common.serialize.ObjectInput;
+import org.apache.dubbo.common.serialize.ObjectOutput;
+import org.apache.dubbo.common.serialize.protostuff.utils.WrapperUtils;
+import org.apache.dubbo.common.utils.ClassHelper;
+import org.apache.dubbo.common.utils.StringUtils;
+import org.apache.dubbo.remoting.Channel;
+import org.apache.dubbo.remoting.exchange.Request;
+
+import javassist.ClassPool;
+import javassist.CtClass;
+import javassist.CtMethod;
+import javassist.LoaderClassPath;
+
+/**
+ * 一个临时解决的办法
+ * 主要描述了问题的产生
+ * 环境
+ * dubbo 2.7.1
+ * protocol 使用 dubbo 序列化设置为 protosubff
+ * 别的为默认
+ * 
+ * 
+ * @author: chenyu
+ * @date:   2019年7月16日 上午10:23:58 
+ *
+ */
+public class ProtosubffPatch {
```

Comment:
would you mind to submit pull request for the issue you found to dubbo: https://github.com/apache/dubbo/pulls